changelog.md

```diff
@@ -5,6 +5,13 @@
 ### Features
 
 - [#2955](https://github.com/ignite/cli/pull/2955/) Add `ignite network request add-account` command.
+- [#2877](https://github.com/ignite/cli/pull/2877) Plugin system
+
```

Comment:
looks like there is some duplication in CHANGELOG now

Reply:
Oups fixed thanks. 

This is because of the that git attribute I added for the changelog merge option here https://github.com/ignite/cli/pull/2814

When the changelog already contains the correct sections before the feature branches are created, this avoids conflicts, but when those sections are added inside differents features branches, this becomes messy. If other people think this is annoying, I can remove it.

Reply:
It seems fine to me other than when we have work that doesn't make it into a release.  I think its ok